ansible/roles/test/files/tools/loganalyzer/loganalyzer_end.yml

```diff
@@ -16,8 +16,12 @@
   shell: grep "TOTAL MATCHES" "{{ test_out_dir }}/{{ summary_file }}" | sed -n "s/TOTAL MATCHES:[[:space:]]*//p"
   register: errors_found
 
+- name: Check if loganalyzer missed expected messages
+  shell: grep "TOTAL EXPECTED MISSING MATCHES" "{{ test_out_dir }}/{{ summary_file }}" | sed -n "s/TOTAL EXPECTED MISSING MATCHES:[[:space:]]*//p"
```

Comment:
Why grep this phrase? Seems it is missing anyway, so it will fail the test case, right?

Reply:
It's not missing. Loganalyzer summary file contains a number of total expected matches, that are missing in logs. We should fail if we didn't get any of expected match.

Reply:
I checked the code: https://github.com/Azure/sonic-mgmt/blob/master/ansible/roles/test/files/tools/loganalyzer/loganalyzer.py#L526
Seems you should grep “TOTAL EXPECTED MATCHES”, right?